src/app/static/src/js/components/runReport/runReportTabs.vue

```diff
@@ -53,10 +54,13 @@
         data() {
             return {
                 selectedTab: "runReport",
-                reportKey: ""
+                selectedRunningReportKey: ""
             }
         },
         methods: {
+            setSelectedReportKey(e){
+                this.selectedRunningReportKey = e
```

Comment:
Do we also need `this.selectedTab = "reportLogs"`?

Reply:
We will do, but I guess that can happen as part of mrc-2150